route/build/cpl/RtmMod.F90

```diff
@@ -283,14 +285,14 @@ SUBROUTINE get_hru_area(NETOPO_in, RPARAM_in, verbose)
         ! Arguments:
         type(RCHTOPO),     intent(in) :: NETOPO_in(:)
         type(RCHPRP),      intent(in) :: RPARAM_in(:)
+        integer, optional, intent(in) :: offset
```

Comment:
I'd like to see a comment to explain what the purpose of offset is

Reply:
added comments on offset